iotics/lib/identity/register/document_helper.py

```diff
@@ -80,9 +134,29 @@ def get_valid_issuer_key_for(doc: RegisterDocument, issuer_name: str, get_contro
     @staticmethod
     def get_valid_issuer_key_for_control_only(doc: RegisterDocument, issuer_name: str,
                                               get_controller_doc: GetControllerDocFunc) -> Optional[IssuerKey]:
+        """
+        Get an IssuerKey matching issuer name (control only)
```

Comment:
`Get a valid issuer key matching issuer name from the control keys and delegation proofs only`